cmd/utils/sub_command.go

```diff
@@ -26,6 +28,8 @@ func SubcommandAction(app SubcommandApplication) cli.ActionFunc {
 		ctx, ctxClose := context.WithCancel(context.Background())
 		defer func() { ctxClose() }()
 
+		rand.Seed(time.Now().UnixNano())
```

Comment:
`rand.Seed` can be called only once, but `SubcommandAction` may not
so we could call `rand.Seed` in `init`

Reply:
I mean more calls with `rand.Seed` doesn't work

Reply:
moved to `main.go`'s `init`: https://github.com/taikoxyz/taiko-client/pull/76/commits/11e48a2c6c1dca8940a8a7e083312dad2f585f07